core/variant_parser.cpp

```diff
@@ -1059,8 +1058,7 @@ Error VariantParser::parse_value(Token &token, Variant &value, Stream *p_stream,
 			}
 
 			value = arr;
-
-		} else if (id == "PoolRealArray" || id == "FloatArray") {
+		} else if (id == "PoolRealArray" || id == "PackedFloat32Array" || id == "PackedFloat64Array" || id == "PackedRealArray" || id == "FloatArray") {
```

Comment:
I don't think we ever use `PackedRealArray` or `PackedIntArray`?

If we did in early 4.0-dev days, there's no need to add compatibility with them.

Reply:
Should we also remove them from master?

Reply:
Ah I missed that they were in `master`. I guess this was for compatibility with early 4.0-dev projects indeed. We might want to keep them for a while in `master` as some 2020 projects might still be around and in need of an update. But eventually we can remove it too there.